wagtail_localize/fields.py

```diff
@@ -2,14 +2,8 @@
 from modelcluster.fields import ParentalKey
 from modelcluster.models import ClusterableModel, get_all_child_relations
 from treebeard.mp_tree import MP_Node
-from wagtail.core.fields import RichTextField, StreamField
-from wagtail.core.models import Page, TranslatableMixin
-
-
-try:
-    from wagtail.core.models import COMMENTS_RELATION_NAME
-except ImportError:
-    COMMENTS_RELATION_NAME = "comments"
```

Comment:
note to self: check the origin of this, as the default for `COMMENTS_RELATION_NAME` is `wagtail_admin_comments`

Reply:
right, so this was a shim for Wagtail < 2.14.2. Safe to drop 🤞